src/App/Pages/Accounts/LoginPasswordlessRequestViewModel.cs

```diff
@@ -277,6 +278,10 @@ private async Task CheckLoginRequestStatus()
                     await HandleLoginCompleteAsync();
                 }
             }
+            catch (ApiException ex) when (ex.Error?.StatusCode == System.Net.HttpStatusCode.BadRequest)
+            {
+                HandleException(ex);
+            }
```

Comment:
🤔 Any `BadRequest` should handle the exception the same or only the ones that corresponds to SSO only policy enforced?

Reply:
I don't think we will have more bad requests other than this one but if the request is bad we don't want to continue to run the check over and over. I'm even leaning towards not having the StartCheck running again in general, it causes more harm that good.

Reply:
Alright I'll approve then and we can revise the start check afterwards.